### PR TITLE
Update SDK page with full list of current SDKs.

### PR DESCRIPTION
**Pull request by markmandel**
Also made a better landing page for the Go SDK

### CODE REVIEW

**Comment by roberthbailey** [comment]
lgtm

File: site/content/en/docs/Guides/Client SDKs/go.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+date: 2019-05-17T10:17:50Z
+weight: 30
+description: "This is the Go version of the Agones Game Server Client SDK. "
+---
+
+Check the [Client SDK Documentation]({{< relref "_index.md" >}}) for more details on each of the SDK functions and how to run the SDK locally.
+
+## Installation
+
+`go get` the source, {{< ghlink href="sdks/go" >}}directly from Github{{< /ghlink >}}
```
Comment (pooneh-m) [inline site/content/en/docs/Guides/Client SDKs/go.md]: This doesn't render correctly in md.

File: site/content/en/docs/Guides/Client SDKs/_index.md [hunk 2]
```diff
@@ -12,8 +12,11 @@ The client SDKs are required for a game server to work with Agones.
 
 The current supported SDKs are:
 
+- [Unreal Engine]({{< relref "unreal.md" >}})
 - [C++]({{< relref "cpp.md" >}})
-- [Go](https://godoc.org/agones.dev/agones/sdks/go)
+- [Node.js]({{< relref "nodejs.md" >}})
+- [Go]({{< relref "go.md" >}})
+- [Rust]({{< relref "rust.md" >}})
```
Comment (pooneh-m) [inline site/content/en/docs/Guides/Client SDKs/_index.md]: How about the local.md?

File: site/content/en/docs/Guides/Client SDKs/go.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Go Game Server Client SDK"
+linkTitle: "Go"
+date: 2019-05-17T10:17:50Z
```
Comment (pooneh-m) [inline site/content/en/docs/Guides/Client SDKs/go.md]: What is this date referring to?

Comment (markmandel) [reply site/content/en/docs/Guides/Client SDKs/go.md]: The date the page got created, that's it.
https://gohugo.io/content-management/front-matter/

In this case, it's not particularly relevant.

File: site/content/en/docs/Guides/Client SDKs/_index.md (hunk 2 above)
Comment (markmandel) [reply site/content/en/docs/Guides/Client SDKs/_index.md]: We have a section below this:

> There is also local development tooling for working against the SDK locally, without having to spin up an entire Kubernetes infrastructure.

That work?

**Pull request merged**